test/Standalone/convolution2d_nhwc_hwcf.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: standalone-opt -pack-conv2DNhwcHwcf="block-factors=32,32" %s | FileCheck %s
```

Comment:
Would also be nice to have a second RUN line that goes all the way to TPP and see if the right GEMM call is being made.

Reply:
I don't have a mapping for this conv.

Reply:
We dont' have for the blocked form: https://github.com/plaidml/tpp-sandbox/blob/dc33e62e9c76f01692bfa8f4c5942ee00e356f7f/lib/Standalone/DecomposeConvsToMatmulOrBrgemm.cpp#L793